src/modules/accumulator.js

```diff
@@ -79,6 +99,46 @@ export default class AccumulatorModule extends WithParamsAndPublicKeys {
     return null;
   }
 
+  /**
+   * Return the accumulated value as hex
+   * @param accumulated {Uint8Array|KBUniversalAccumulatorValue}
+   * @param typ {number} - Type of the accumulator
+   * @returns {string}
+   */
+  static accumulatedAsHex(accumulated, typ = AccumulatorType.VBPos) {
```

Comment:
I would be better to define all accumulated variants as classes like it's done with `KBUniversalAccumulatorValue `.
They can extend a class that has `fromHex`/`toHex` methods and encapsulate all this logic.

Reply:
Thanks. Done

Reply:
It would be better to make this change in the TS dependency so i'll leave it like this here.